app/src/main/java/org/kiwix/kiwixmobile/help/HelpActivity.java

```diff
@@ -66,13 +67,16 @@ protected void onCreate(Bundle savedInstanceState) {
   @OnClick({ R.id.activity_help_feedback_text_view, R.id.activity_help_feedback_image_view })
   void sendFeedback() {
     Intent intent = new Intent(Intent.ACTION_SENDTO);
-    intent.setType("plain/text");
-    String uriText = "mailto:" + Uri.encode(CONTACT_EMAIL_ADDRESS) +
-        "?subject=" + Uri.encode(
-        "Feedback in " + LanguageUtils.getCurrentLocale(this).getDisplayLanguage());
-    Uri uri = Uri.parse(uriText);
-    intent.setData(uri);
-    startActivity(Intent.createChooser(intent, "Send Feedback via Email"));
+    intent.setData(Uri.parse("mailto:"));
+    intent.putExtra(Intent.EXTRA_EMAIL,new String[]{ CONTACT_EMAIL_ADDRESS });
+    intent.putExtra(Intent.EXTRA_SUBJECT, "Feedback in " + LanguageUtils.getCurrentLocale(this).getDisplayLanguage());
+
+    try {
+      startActivity(Intent.createChooser(intent, "Send Feedback via Email"));
+    }
+    catch(android.content.ActivityNotFoundException ex){
```

Comment:
`catch` should be on the previous line. Follow the contributors' guidelines. Import the Square's code style and reformat your code using `Ctrl + Shift + L` to avoid such mistakes.